cmake/cudawrappers-helper.cmake

```
@@ -7,8 +7,10 @@ function(inline_local_includes input_file output_file)
   string(REGEX MATCHALL ${include_regex} includes ${input_file_contents})
   set(include_files "")
   foreach(include ${includes})
-    string(REGEX REPLACE ${include_regex} "\\3" include_name ${include}) # e.g. 'helper.h'
-    string(REGEX REPLACE ${include_regex} "\\2" include_line ${include}) # e.g.  '#include <helper.h>'
+    string(REGEX REPLACE ${include_regex} "\\3" include_name ${include}
+    )# e.g. 'helper.h'
+    string(REGEX REPLACE ${include_regex} "\\2" include_line ${include}
+    )# e.g.  '#include <helper.h>'
```

I don't like the auto-formatting in this case. How about changing it to:
```suggestion
    # Get the name of the file to include, e.g. 'helper.h'
    string(REGEX REPLACE ${include_regex} "\\3" include_name ${include})
    # Get the complete line of the include, e.g.  '#include <helper.h>'
    string(REGEX REPLACE ${include_regex} "\\2" include_line ${include})
```